src/libsystemd-network/dhcp-option.c

```diff
@@ -48,6 +49,32 @@ static int option_append(uint8_t options[], size_t size, size_t *offset,
                 *offset += 1;
                 break;
 
+        case SD_DHCP_OPTION_USER_CLASS: {
+                unsigned i = 0;
+                char **s;
+
+                STRV_FOREACH(s, optval) {
```

Comment:
i figure you dropped the "const" from the function parameter, so that this STRV_FOREACH doesn't complain about it? 

Note that we can't use const on strv, even if they are constant, due to C's rules regarding const on ptrs to ptrs... But I think this should be an implementation detail here, hence I think we should leave the function parameter const but then cast this away as needed...